react/features/recording/functions.ts

```diff
@@ -172,8 +172,9 @@ export function getRecordButtonProps(state: IReduxState) {
 
     const dropboxEnabled = isDropboxEnabled(state);
 
-    visible = isModerator && (recordingService?.enabled || localRecordingEnabled || dropboxEnabled);
-    visible = isJwtFeatureEnabled(state, 'recording', visible);
+    visible = isModerator
+        ? isJwtFeatureEnabled(state, 'recording', recordingService?.enabled || localRecordingEnabled || dropboxEnabled)
```

Comment:
Same and here, the last param of isJwtFeatureEnabled is used only when there is no token. But those are general configs for the service and they should overwrite the decision from isJwtFeatureEnabled.